assopy/admin.py

```diff
@@ -221,7 +221,7 @@ def do_edit_invoices(self, request, queryset):
 
     def edit_invoices(self, request):
         try:
-            ids = map(int, request.GET['id'].split(','))
+            ids = list(map(int, request.GET['id'].split(',')))
```

Comment:
maybe we can replace that with list comprehension in the same PR?

Reply:
yup